content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -105,11 +113,15 @@ The :guilabel:`Vendor Reference` field points to the sales and delivery order nu
 vendor. This comes in handy once products are received, and the |PO| needs to be matched to the
 delivery order.
 
-The :guilabel:`Blanket Order` field refers to long-term purchase agreements on recurring orders with
-set pricing. To view and configure blanket orders, head to :menuselection:`Purchase app --> Orders
---> Purchase agreements`.
+With the :doc:`Purchase Agreements feature <blanket_orders>` activated, the :guilabel:`Blanket
```

Comment:
I like that we're talking about blanket orders and dropshipping, but at this location in the content, it seems a bit premature since we're simply just trying to create an RFQ? We don't need any of these features enabled in order to create an RFQ...